en/components/themes/themes-list/fluent.md

```diff
@@ -0,0 +1,24 @@
+## Switching from Material to Fluent Theme
+Since version `8.2` our components include a new theme inspired by the [`Microsoft Fluent Design System`](https://www.microsoft.com/design/fluent/)  
+  
+We also added a `Microsoft Word` palette. To use it just replace the `$fluent-excel-palette` with `$fluent-word-palette`:  
```

Comment:
Say something like we support Word and Excel palette. You have not introduced the Excel palette to say that we also have a Word one and it sounds weird.